tests/Silex/Tests/ApplicationTest.php

```diff
@@ -441,7 +441,7 @@ public function testRegisterShouldReturnSelf()
     public function testMountShouldReturnSelf()
     {
         $app = new Application();
-        $mounted = new ControllerCollection(new Route());
+        $mounted = new ControllerCollection($app, new Route());
```

Comment:
As the new argument is now optional, the tests can be keep like before, but new tests should be added to cover the new feature.